src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC } from 'react';
+import cn from 'classnames';
+
+import { Todo } from './../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  selectedTodo: Todo | null;
+  onSelectTodo: (todo: Todo) => void;
+}
+
+export const TodoItem: FC<Props> = ({ todo, selectedTodo, onSelectTodo }) => {
+  const { id, title, completed } = todo;
+
+  return (
+    <tr data-cy="todo" className="">
```

Comment:
It seems like you're using an empty className here. If you don't need to apply any class to this element, you can just omit the className attribute.